tools/eslint-rules/rules/valid-schema-description.ts

```diff
@@ -24,7 +24,7 @@ export const rule = ESLintUtils.RuleCreator(() => __filename)({
   defaultOptions: [],
   create(context) {
     // jsonc-eslint-parser adds this property to parserServices where appropriate
-    if (!(context.parserServices as any).isJSON) {
+    if (!(context.sourceCode.parserServices as any).isJSON) {
```

Comment:
We don't need to wrap this here, it is not a shared rule, so we just migrate the usage